composer.json

```diff
@@ -17,20 +17,27 @@
         }
     ],
     "require": {
-        "php": ">=5.3.6",
+        "php": "^7.1",
         "behat/behat": "^3.0",
         "behat/symfony2-extension": "^2.0",
-        "doctrine/data-fixtures": "^1.0"
+        "doctrine/data-fixtures": "^1.0",
+        "doctrine/doctrine-bundle": "^1.8",
+        "doctrine/doctrine-fixtures-bundle": "^3.0",
+        "doctrine/orm": "^2.6",
+        "symfony/process": "^4.0"
```

Comment:
Can we fallback to latest LTS at least? `^3.4` would be useful for more users. For me it's a showstopper to test for now

Reply:
Definetely a showstoper, since assetic requires process ~2|~3 and I even cannot override single process dependency for testing purposes

Reply:
Thank you for that feedback. I updated that dependency accordingly.